flink-connectors/flink-hadoop-compatibility/src/test/scala/org/apache/flink/api/hadoopcompatibility/scala/WordCountMapreduceITCase.scala

```diff
@@ -1,91 +0,0 @@
-/*
```

Comment:
The WordCountMapreduceITCase has a corresponding Java version, so delete it directly.